--- core/sync/build.gradle.kts
+++ core/sync/build.gradle.kts
@@ -3,6 +3,7 @@
     id("org.jetbrains.kotlin.android")
     id("com.google.dagger.hilt.android")
     id("com.google.devtools.ksp")
+    id("org.jetbrains.kotlin.plugin.serialization")
 }

 android {
@@ -36,6 +37,7 @@
     ksp(libs.androidx.hilt.compiler)

     implementation(libs.androidx.datastore.preferences)
+    implementation(libs.kotlinx.serialization.json)

     testImplementation(libs.junit)
     testImplementation(libs.robolectric)
